BugByte-backend/src/main/java/com/example/BugByte_backend/repositories/PostingRepository.java

```diff
@@ -397,9 +463,10 @@ public Boolean verifyAnswer(Long answerId, Long questionId) {
         if(answerId == null)
             throw new NullPointerException("answer id is null");
 
-        int rows = jdbcTemplate.update( SQL_VERIFY_ANSWER, answerId, questionId);
+        int rows = jdbcTemplate.update(SQL_VERIFY_ANSWER, answerId, questionId);
+        int users_rows = jdbcTemplate.update(SQL_UPDATE_REPUTATION_VERIFIED_ANSWER, answerId);
```

Comment:
The current implementation does not account for scenarios where the verified answer is either removed or reassigned to another answer. This can result in multiple users incorrectly receiving reputation points for the verified answer. Only a single user should receive points for the verified answer at any given time. Please revise the logic to address these edge cases.

Reply:
When verifying an answer, we should ensure it's correct before marking it. If the verification was a mistake, we need also to remove it from the "verified answers" column  (i'm not sure if we do).
but it should be removed from reputation only in case of accedintally verification.